client/src/pages/Login/Login.tsx

```diff
@@ -36,25 +36,37 @@ export default function Login(): JSX.Element {
     });
   };
 
+  const [open, setOpen] = React.useState(true);
+
+  const handleOpenLoginModal = () => {
+    setOpen(true);
+  };
+
+  const handleCloseLoginModal = () => {
+    setOpen(false);
+  };
+
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
-        <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
-                </Typography>
-              </Grid>
+      <Dialog onClose={handleCloseLoginModal} open={open} disableBackdropClick={true} disableEscapeKeyDown={true}>
```

Comment:
try adding the same minWidth to your dialog components through a className to see if they match

Reply:
I'll try it

Reply:
It didn't work
But I found I could do it in theme.ts by changing the properties of the paper component of the dialog